src/plugins/intel_gna/transformations/transpose_nchw.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace GNAPluginNS {
+
+/**
+ * @brief TODO
+ */
+class SubstituteGNAConvolution : public ngraph::pass::MatcherPass {
+public:
+    NGRAPH_RTTI_DECLARATION;
+    SubstituteGNAConvolution();
+};
+
+class SubstituteGNAMaxPool : public ngraph::pass::MatcherPass {
+public:
+    NGRAPH_RTTI_DECLARATION;
+    SubstituteGNAMaxPool();
+};
+
+class TransposeNCHW : public ngraph::pass::FunctionPass {
```

Comment:
from my point of view "TransposeNCHW" isn't quite clear name. I'd reflect in the name what exactly is transposed and to which layout.